en/docs/assets/css/config-catalog.css

```
@@ -0,0 +1,288 @@
+    border-bottom: 1px solid #eee;
+}
+
+.mb-config-catalog>section.title>.mb-config-example {
+    display: none;
+}
+
+.mb-config-catalog .mb-config-example code,
+.mb-config-catalog .mb-config-example pre {
+    margin: 0;
+    white-space : pre-wrap !important;
+    word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)